dist/metricsgraphics.css

```diff
@@ -5,6 +5,10 @@
     opacity: 0.8;
 }
 
+.mg-area-color {
+    fill: #000000;
```

Comment:
You should be able to get away with just `#000` here (and below).

Reply:
Done